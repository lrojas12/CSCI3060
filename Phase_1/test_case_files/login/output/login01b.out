Welcome to Watermelon Banking System. Please login to begin or type in “help” for more information.
Mode in which you wish to log in as: 
Account holder’sname: 
You are currently logged in as acc_holder
BankAccount Number: acc_num
Balance: balance
Transaction PaymentPlan: plan
Status: status 
You have successfully logged out of your account.