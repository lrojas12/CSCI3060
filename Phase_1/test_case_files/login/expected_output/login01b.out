Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to log in as: 
Account holder’s name: 
You are currently logged in as Tarzan.
BankAccount Number: 00001
Balance: $01000.00
Transaction PaymentPlan: Non-student
Status: Active
You have successfully logged out of your account.